spec/import_spec.rb

```diff
@@ -158,9 +161,10 @@ def upload_file!(name, ext='csv')
 
       context "without headers" do
         context "with known csv headers" do
-          let(:options) {
-            {template_object: ActiveAdminImport::Model.new(csv_headers: ['Name', 'Last name', 'Birthday'])}
-          }
+          
+          let(:options) do
+            { template_object: ActiveAdminImport::Model.new(csv_headers: ['Name', 'Last name', 'Birthday']) }
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.